bigtop-packages/src/common/sqoop/patch2-SQOOP-3474.diff

```diff
@@ -0,0 +1,21 @@
+diff --git a/ivy.xml b/ivy.xml
+index e4b45bfd..63cdcce4 100644
+--- a/ivy.xml
++++ b/ivy.xml
+@@ -103,6 +103,8 @@ under the License.
+       conf="common->default;redist->default"/>
+     <dependency org="commons-io" name="commons-io" rev="${commons-io.version}"
+       conf="common->default;redist->default"/>
++    <dependency org="commons-lang" name="commons-lang" rev="${commons-lang.version}"
++      conf="common->default;redist->default"/>
+     <dependency org="org.apache.commons" name="commons-lang3" rev="${commons-lang3.version}"
+       conf="common->default;redist->default"/>
+     <dependency org="org.kitesdk" name="kite-data-mapreduce" rev="${kite-data.version}"
+@@ -121,8 +123,6 @@ under the License.
+
+     <dependency org="com.google.code.p.arat" name="rat-lib"
+           rev="${rats-lib.version}" conf="releaseaudit->default" />
+-    <dependency org="commons-lang" name="commons-lang"
+-      rev="${commons-lang.version}" conf="releaseaudit->default"/>
+     <dependency org="commons-collections" name="commons-collections"
+       rev="${commons-collections.version}" conf="releaseaudit->default"/>
```

Comment:
Could you rename the patch to the name like `patch2-redist-commons-lang.diff`?  We usually mark the patch with JIRA id if it is the one submitted/committed/backported. There is no patch submitted to SQOOP-3474 and it was originally reported as issue of sqoop2-client.